packages/component-boilerplate/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@saasquatch/component-boilerplate",
+  "version": "1.0.0-13",
+  "private": false,
+  "description": "Boilerplate for writing web components for the SaaSquatch widget environments",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "exports": "./dist/index.modern.js",
+  "module": "dist/index.module.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "microbundle",
+    "test": "jest",
+    "test-watch": "jest --watch"
+  },
+  "author": "SaaSquatch",
+  "license": "ISC",
+  "devDependencies": {
+    "@saasquatch/universal-hooks-testing-library": "^0.1.0",
+    "@testing-library/react-hooks": "^5.1.0",
+    "@types/debounce": "^1.2.0",
+    "@types/debug": "^4.1.5",
+    "@types/jest": "^26.0.20",
+    "axios": "^0.21.1",
+    "esbuild-jest": "^0.5.0",
+    "jest": "^26.6.3",
+    "jest-cucumber": "^3.0.1",
+    "microbundle": "^0.13.0",
+    "msw": "^0.28.1",
+    "react": "^17.0.1",
+    "react-test-renderer": "^17.0.1",
+    "ts-jest": "^26.5.4",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.1.5"
+  },
+  "dependencies": {
+    "@formatjs/intl": "^1.8.2",
+    "@saasquatch/dom-context-hooks": "^0.1.0",
+    "@saasquatch/universal-hooks": "^0.1.3",
```

Comment:
These should be the 1.0 versions

Reply:
i'll set up a dom-context-hooks 1.0 PR as well

Reply:
@loganvolkers i don't have write access on the `dom-context-hooks` repo